frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/ERDContent.tsx

```diff
@@ -35,13 +37,27 @@ type Props = {
     | undefined
 }
 
+export const isRelatedToTable = (
+  relationships: Relationships,
+  tableName: string,
+  targetTableName: string | undefined,
+) =>
+  Object.values(relationships).some(
```

Comment:
suggestion:

```suggestion
  targetTableName: string | undefined,
) =>
  if (!targetTableName) return
  Object.values(relationships).some(
```

might be faster if targetTableName is undefined often .

Reply:
That's right! Thank you!🙏
Fixed: https://github.com/liam-hq/liam/pull/221/commits/db4dbc262b36b9acbca9f0e656a8028a8fb55d84